src/harness/dtRunner.ts

```diff
@@ -0,0 +1,62 @@
+/// <reference path="harness.ts"/>
+/// <reference path="runnerbase.ts" />
+class DefinitelyTypedRunner extends RunnerBase {
+    private static readonly testDir = "../DefinitelyTyped/types/";
+
+    public workingDirectory = DefinitelyTypedRunner.testDir;
+
+    public enumerateTestFiles() {
+        return Harness.IO.getDirectories(DefinitelyTypedRunner.testDir);

```

Comment:
do you want to add an assert here or something to let users know that DefinitllyTyped was not found in the expected place instead of silently not running any tests.